cmake/modules/CMakeFunctions.cmake

```diff
@@ -39,6 +39,51 @@ macro(use_eh val)
     endif()
 endmacro(use_eh)
 
+# Reads hash-commit from backported patch
+# This function assumes that each of files starts with (for example):
+# From 1a400928bf8fc86fa0f062524c25d0985c94ac6f Mon Sep 17 00:00:00 2001
+function(get_backport_patch_hash patch_path patch_hash)
```

Comment:
can we use "git rev-parse " to get commit id?

Reply:
I cannot see how I could read sha1 using git rev-parse from the file containing the patch before patching? Could you share the exact command line which you are thinking?

Reply:
I probably did a wrong test, rev-parse is not working.